docs/docs/migrate-from-v1/README.md

```diff
@@ -0,0 +1,319 @@
+  async run({ steps, $ }) {
+    $.respond({
+      status: 200,
+      headers: {},
+      body: { 
+        message: "hello world!"
+      }
+    });
+  },
+})
+```
+
+Please note, you'll also need to configure the HTTP trigger step to also allow custom responses. Use the dropdown in the **HTTP Response** section of the HTTP trigger to select the **Return a custom response from your workflow** option:
+
+<img src="./images/custom-http-response-option.png" alt="Select the option to allow your workflow to send it's own HTTP responses">
+
+## Known Gaps & Limitations
+
+However, some features from the original builder are not currently available in v2. The Pipedream team is working to quickly address these items, but if you have feedback that isn't listed here, please [[[reach out](/support).
+
+### Sharing workflows
+
+At this time, sharing is not yet implemented in v2 of the workflow builder.
+
+We're working on bringing this same feature to the new version. If you need assistance transferring workflows across accounts, [please contact us](/docs/support).
+
+### `$checkpoint`
+
+The `$checkpoint` functionality to save data between workflow runs has been removed.
+
+But you can leverage the `$.service.db` service to store arbitrary data across your workflow runs like unique IDs. [Read more about using `$.database` here.](https://pipedream.com/docs/code/nodejs/#managing-state)
+
+::: warning
+Please note that any values stored in `$.service.db` are only accessible in subsequent workflow runs _in the same step_.
+:::
+
+### Public workflows
+
+At this time, all v2 workflows are private.
+
+If you're working with Pipedream support to troubleshoot your workflow, you can share it with the support team under your workflow's **Settings**.
+
+### Versioned deployments and rollback
+
+In v2, you can test and save your progress on a workflow _without_ deploying it.
+
+However, after deploying it's not possible to rollback to a prior version.
+
+## FAQs
+
+### What are the benefits of the new (v2) workflow builder?
+
+* **Edit & test** your workflows in separate editing mode without impacting live workflows
+* **Support for multiple languages** including Node, Python, Golang & bash
+* **Granular testing** including the ability to test individual steps and more
+* **Multiple triggers** are now supported per workflow
+* **Improved** forms for easier configuration and streamlined building
+
+### What are the limitations of the new (v2) workflow builder?
+
+* `$checkpoint` has been removed from v2 workflows, but `$.database` is a near equivalent.
+* Sharing workflows is not supported
+* Making workflows public is not supported
+* Workflows are no longer versioned
+
+[The limitations of the new workflow builder are covered in detail here](#limitations).
+
+### Are v2 workflows backwards compatible?
+
+If your workflow is not using custom Node.js code and is compromised of only prebuilt actions, then there's a good chance it's supported.
```

Comment:
Since people don't really know the distinction between v1 and v2 actions, I don't think we should say this. We're still a bit of a ways out to get consistent v2 action coverage.

Maybe you can state the facts of the situation plainly for users and encourage them to reach out if they observe specific gaps, e.g. you can say that we only support component actions in the new builder and that we're working quickly to migrate all actions over. If you notice a missing action, please reach out.